jest.config.js

```diff
@@ -42,4 +42,7 @@ module.exports = {
 			'react-native-svg',
 		],
 	},
+	transformIgnorePatterns: [
+		'node_modules/(?!(simple-html-tokenizer|react-native|react-native-svg|react-native-recyclerview-list|react-native-modal|react-native-animatable)/)',
```

Comment:
Cool! what do you think about maybe adding a comment here, as a pointer for people in the future to know they should add other subprojects here as they add them?

Reply:
So for the record, the reason we need this is because gutenberg introduces a non-standard import at some point to avoid importing large entities defined in a submodule: https://github.com/WordPress/gutenberg/pull/12072/files#diff-a252d92fe101838128a4480faa50e745L4
This specific module `simple-html-tokenizer`, produces a single build file (`index.js) when compiled to ES2015 which made the following import
```
import Tokenizer from 'simple-html-tokenizer/dist/es6/tokenizer';
```
fail in gutenberg-mobile. Switching to using the es6 version of the module required some modifications in our jest config.